src/roles/claimer.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const MetaRole = require('roles_meta')
+
+class Claimer extends MetaRole {
+  constructor () {
+    super()
+    this.defaultEnergy = 850
+  }
+
+  getBuild (room, options) {
+    this.setBuildDefaults(room, options)
+    return Creep.buildFromTemplate([MOVE, CLAIM, MOVE, MOVE, MOVE, MOVE], options.energy)
```

Comment:
Shouldn't the CLAIM part be further toward the back of the body, or is this in reverse order?

Reply:
The issue is that our build system doesn't reorder parts, it adds them in one at a time until it hits the max energy. I want the system to be able to send out claimers as soon as it can support them, which means 650 energy. But if I put the claim part last it would require 850 energy before we could build claimers.

Ultimately I think a future PR should modify the creep build system to move parts around (shoving all the tough in the front for instance, as well as all the claims to the back).

Reply:
So in other words, yes it is in reverse order. :) All good then!
But yes, making sure all tough parts are in front is important. I second that notion.

Reply:
I don't know what you mean by reverse order then. It will make a 1M1C creep at it's smallest, and a 1M1C4M creep at it's largest.

Reply:
Oh. I see. In that case the thing to sort/reorder bodyparts would be a bigger improvement than I thought. :)